docs/user/status.md

```diff
@@ -117,9 +117,9 @@ It could also be caused by response corruption, e.g. if the configured `upstream
 
 ### VersionNotFound
 
-The currently installed version was not found in the configured `channel`.
+The currently reconciling version was not found in the configured `channel`.
```

Comment:
Suggestion: `The version CVO is reconciling not found in the configured channel`.

Reply:
I'm fine with both (the first one sounds less complex though)

Reply:
I would rather not use the CVO acronym in user-facing condition messages (which this doc entry just echos).  If you feel like my wording here is ambiguous, how about:

> The currently reconciling cluster version was not found in the configured `channel`.

Reply:
`The currently reconciling cluster version was not found in the configured channel` sounds good to me.

Reply:
Also intent was to be more explicit. Not using `CVO` as an acronym is a good idea. However the doc already have `CVO` in the text.

Reply:
This line appears in both the doc and the condition message.  The CVO references should appear only in the doc.  Will update In a bit.

Reply:
Rerolled to use "currently reconciling cluster version" with 9f15457f -> ea6899ae.